packages/terra-clinical-item-collection/src/ItemCollection.scss

```diff
@@ -1,10 +1,31 @@
 :local {
-  .column-accessory,
-  .list-accessory {
+  .table {
+    table-layout: fixed;
+  }
+
+  .start-accessory-column {
     width: 40px;
   }
 
-  .content-accessory {
+  .end-accessory-column {
+    width: calc(100% / 7);
```

Comment:
I talked offline with @emilyrohrbough regarding this width for the end accessory column. It provides the end accessory column with more real estate than I'd expect in the examples, and I'm not sure we can land on a fixed value here that will satisfy the majority of implementations.

Emily suggested giving the end accessory the same width as the other text/comment columns, which I think makes sense. If the end accessory column width ever needs to be customizable or dynamic, we should investigate adding custom column width support across the entire table like we've talked about before. Until then, I think it makes sense to be as un-opinionated as we can be.